woocommerce/woocommerce
# Add `@types/*` & declare TS modules for woocommerce-admin

### All Submissions:






### Changes proposed in this Pull Request:



Part of https://github.com/woocommerce/woocommerce/issues/32581

Add `@types/testing-library__jest-dom` & declare as TS modules for non-TS modules to fix admin TS errors:

- Property 'toBeInTheDocument' does not exist on type 'JestMatchers<HTMLElement | null>'.
- Could not find a declaration file for module '@wordpress/plugins'.
- Could not find a declaration file for module '@woocommerce/tracks'.
- Cannot find module './logo.png' or its corresponding type declarations.

### How to test the changes in this Pull Request:

1. Run `pnpm install & pnpm nx build woocommerce-admin`
2. Run `pnpm nx ts:check woocommerce-admin`
3. Should not see above TS errors

### Other information:




### FOR PR REVIEWER ONLY:
